src/test/java/org/example/integration/JobRoleManagerIntegrationTest.java

```diff
@@ -0,0 +1,31 @@
+package org.example.integration;
+
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.example.JobRoleManagerApplication;
+import org.example.JobRoleManagerConfiguration;
+import org.example.models.JobRoleResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Client;
+import java.util.List;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class JobRoleManagerIntegrationTest {
+    private static final DropwizardAppExtension<JobRoleManagerConfiguration> APP =
+            new DropwizardAppExtension<>(JobRoleManagerApplication.class);
+
+
```

Comment:
I think we need Test annotation back. Disabled is the annotation to skip it, and you can write a comment as to why

[`@Disabled `](https://junit.org/junit5/docs/5.2.0/api/org/junit/jupiter/api/Disabled.html)